### ISSUE TITLE
Implement a search feature that allows users to find other users or posts by keywords

### ISSUE THREAD

**Issue by Umang375** [Umang375/Real_Social#1]
There is a search box on the navbar component and when I enter the name or a string it should open a modal containing all the suggestions or Not Found if the user searched does not exist.

**Comment by H1s1** [comment]
Hey @Umang375, I would like to add the search functionality. Could you please assign me this issue? Since I am new to open source, your suggestions and guidance are appreciated

**Comment by Umang375** [comment]
Have you seen how the database is set in the image? Checkout the contributing.md

**Comment by H1s1** [comment]
Yes, I saw that image on the Google Drive link. Please guide me on what I should do next?

**Comment by Umang375** [comment]
The project is made using a simple create-react-app, I expect you have worked with full-stack projects before. The DB used is MySQL you can use any client I recommend using SQL workbench from Oracle and make the table and copy the foreign keys the same way mentioned in the image. After this, you need to set the project you can easily set it using npm install command backend is a folder named API and the frontend is among public and src library
